inspirehep/submissions/views.py

```diff
@@ -156,6 +165,151 @@ def start_workflow_for_submission(self, submission_data, control_number=None):
         abort(503)
 
 
+class JobSubmissionsResource(BaseSubmissionsResource):
+    decorators = [login_required_with_roles()]
+    user_allowed_status_changes = {
+        "pending": ["pending"],
+        "open": ["open", "closed"],
+        "closed": ["closed"],
+    }
+
+    def get(self, pid_value):
+        try:
+            record = JobsRecord.get_record_by_pid_value(pid_value)
+        except PIDDoesNotExistError:
+            abort(404)
+
+        serialized_record = job_v1.dump(record)
+        return jsonify({"data": serialized_record})
+
+    def post(self):
+        """Adds new job record"""
+        data = job_loader_v1()
+        data["status"] = "pending"
+        data = self.prepare_data(data)
+        record = JobsRecord.create(data)
+        record.commit()
+        db.session.commit()
+        self.create_ticket(record)
+        response = jsonify({"status": "created"})
+        response.status_code = 201
+        response.headers["location"] = url_for(
```

Comment:
do you mean ``Link``? http://www.rfc-editor.org/rfc/rfc5988.txt

Reply:
but still I don't see any value for this right now, we are not using using it anywhere, and it will be nicer if we envelop the response in something like:

```json
{
   "links": {},
   "data": {}
}
```